.github/workflows/test-changed-auth.yml

```diff
@@ -7,32 +7,62 @@ env:
+  test-chrome:
+    name: Test Auth on Chrome and Node If Changed
     runs-on: ubuntu-latest
 
     steps:
-    # install Chrome first, so the correct version of webdriver can be installed by chromedriver when setting up the repo
-    - name: install Chrome stable
-      run: |
-        sudo apt-get update
-        sudo apt-get install google-chrome-stable
-    - name: Checkout Repo
-      uses: actions/checkout@master
-      with:
-        # This makes Actions fetch all Git history so run-changed script can diff properly.
-        fetch-depth: 0
-    - name: Set up Node (14)
-      uses: actions/setup-node@v2
-      with:
-        node-version: 14.x
-    - name: Bump Node memory limit
-      run: echo "NODE_OPTIONS=--max_old_space_size=4096" >> $GITHUB_ENV
-    - name: Test setup and yarn install
-      run: |
-        cp config/ci.config.json config/project.json
-        yarn
-    - name: build
-      run: yarn build:changed auth
-    - name: Run tests on changed packages
-      run: xvfb-run yarn test:changed auth
\ No newline at end of file
+      # install Chrome first, so the correct version of webdriver can be installed by chromedriver when setting up the repo
+      - name: install Chrome stable
+        run: |
+          sudo apt-get update
+          sudo apt-get install google-chrome-stable
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          # This makes Actions fetch all Git history so run-changed script can diff properly.
+          fetch-depth: 0
+      - name: Set up Node (14)
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - name: Bump Node memory limit
+        run: echo "NODE_OPTIONS=--max_old_space_size=4096" >> $GITHUB_ENV
+      - name: Test setup and yarn install
+        run: |
+          cp config/ci.config.json config/project.json
+          yarn
+      - name: build
+        run: yarn build:changed auth
+      - name: Run tests on changed packages
+        run: xvfb-run yarn test:changed auth
+  test-firefox:
+    name: Test Auth on Firefox If Changed
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: install Firefox stable
+        run: |
+          sudo apt-get update
+          sudo apt-get install firefox
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          # This makes Actions fetch all Git history so run-changed script can diff properly.
+          fetch-depth: 0
+      - name: Set up Node (14)
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - name: Bump Node memory limit
+        run: echo "NODE_OPTIONS=--max_old_space_size=4096" >> $GITHUB_ENV
+      - name: Test setup and yarn install
+        run: |
+          cp config/ci.config.json config/project.json
+          yarn
+      - name: build
+        run: yarn build:changed auth
+      - name: Run tests on changed packages
+        run: xvfb-run yarn --cwd packages/auth test:browser:unit
```

Comment:
It looks like test:changed auth would have run tests in both auth and auth-compat so in order to match, auth-compat should probably be run as well.

Reply:
Updated to accommodate compat